data.js

```diff
@@ -9,7 +9,7 @@ const hardcodedData = {
     startingN: 500,
     endingN: 95500,
     resolution: 5000,
-    graphFilename: null,
+    graphFilename: 'graph-images/bubbleSort.algo.png',
```

Comment:
We may want to consider a `path.resolve()` solution going forward. But, this is perfect for this iteration.